rpc/backend/chain_info.go

```diff
@@ -303,7 +303,7 @@ func (b *Backend) FeeHistory(
 // Although we don't support tx prioritization yet, but we return a positive value to help client to
 // mitigate the base fee changes.
 func (b *Backend) SuggestGasTipCap(baseFee *big.Int) (*big.Int, error) {
-	if baseFee == nil {
```

Comment:
this too, we should change the calculation below to use `big.Int`, similar to the `x/feemarket/keeper/eip1559.go`

Reply:
actually it's impossible to have Int64() out of bound here

Reply:
the base fee won't exceed 10cro?

Reply:
big.Int use Int64() is fine, math.Int is not

Reply:
it'll truncate silently? technically, that's not correct though.